rust/src/utils.rs

```diff
@@ -0,0 +1,232 @@
+use anyhow::bail;
+use axiom_codec::constants::USER_MAX_OUTPUTS;
+use axiom_codec::types::native::{AxiomV2ComputeQuery, AxiomV2ComputeSnark};
+use axiom_eth::halo2_base::utils::ScalarField;
```

Comment:
nit: I prefer when the imports are grouped together. VS Code starts to auto un-group once there's two that are ungrouped.